cmd/veneur/main.go

```diff
@@ -187,6 +189,15 @@ func main() {
 	if err != nil {
 		logger.WithError(err).Fatal("Could not initialize server")
 	}
+
+	go func() {
+		flushIntervalInMilliseconds := int64(server.Interval) / int64(math.Pow(10, 6))
```

Comment:
`server.Interval` is a `time.Duration` so you can use `time.Milliseconds()` for this instead of needing to hardcode a constant

Reply:
Also not sure why we convert to milliseconds, then away from milliseconds